mods/ra/rules/vehicles.yaml

```diff
@@ -563,7 +563,7 @@ TTNK:
 	Armament:
 		Weapon: TTankZap
 		LocalOffset: 0,0,213
-	AttackFrontal:
+	AttackOmni:
```

Comment:
This doesn't work at all for mobile units. Try ordering the tesla tank to a unit that's further away than its weapon range: it won't move towards the target at all.